proposals/lip-0045.md

```diff
@@ -1411,15 +1411,41 @@ def getTerminatedOutboxAccount(chainID: ChainID) -> TerminatedOutboxAccount:
     return terminatedOutboxAccount(chainID)
 ```
 
+### Endpoints for Off-Chain Services
+
 #### isChainIDAvailable
 
 ```python
 def isChainIDAvailable(chainID: ChainID) -> bool:
-    if chainAccount(chainID) does not exist:
-        return True
-    return False
+    if chainAccount(chainID) exist:
```

Comment:
A minor point, but I would move this check to the end, as it requires accessing the state (expensive) while other checks don't (cheap)